res/values-de/strings.xml

```diff
@@ -260,4 +260,8 @@
 	<string name="one_year_ago">vor 1 Jahr</string>
 	<string name="n_years_ago">vor %d Jahren</string>
 
+	<!-- Warning strings -->
+	<string name="warn_thread_archived_toast">Kann nicht auf archiviertem Thread abstimmen.</string>
```

Comment:
How were these translated?  If they weren't written by a native speaker, it may make more sense to leave them out for now and let TalviT add them in a separate PR.

Reply:
Google translate. 
I haven't tested internationalization before, so I don't know how it works if you try to look up a string that is missing a translation.

Reply:
Good to err on the side of fewer bugs :)  Fortunately it just gracefully defaults to whatever's in the plain res/values directory.

Let's revert the German changes for now and once these PRs are all tested and merged I'll ping TalviT so they can localize the new strings whenever they have the time.

Reply:
I can do that when I get home.

Reply:
Should I just leave them in the file but commented out? Should I leave them in the file as English? Or should I just remove them entirely?

Reply:
You can simply revert all changes to files in values-de.